docs/sources/alerting/configure-notifications/manage-contact-points/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
 aliases:
-  - ../contact-points/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/
```

Comment:
I checked that there is still a redirect in place, I think this removal is fine because it there is still a redirect in that takes /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/ 
 to docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/notifications/contact-points/

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344